code_samples/back_office/content_type/templates/my_template.html.twig

```diff
@@ -0,0 +1,12 @@
+{% extends '@ibexadesign/ui/component/anchor_navigation/primary_section.html.twig' %}
+
+{% set data_id = 'ibexa-edit-content-sections-tags' %}
+
+{% block sections %}
+    {% embed '@ibexadesign/ui/component/anchor_navigation/secondary_section.html.twig' %}
+        {% set data_id = 'ibexa-edit-content-sections-item-SEO' %}
+        {% block content %}
+            Contents of 'SEO' secondary section
+        {% endblock %}
+    {% endembed %}
+{% endblock %}
```

Comment:
`data_id` variable has to match `data-target-id` attribute from the Listener code above. I fixed it for you:

```suggestion
{% extends '@ibexadesign/ui/component/anchor_navigation/primary_section.html.twig' %}

{% set data_id = 'ibexa-edit-content-sections-new-tab' %}

{% block sections %}
    {% embed '@ibexadesign/ui/component/anchor_navigation/secondary_section.html.twig' %}
        {% set data_id = 'ibexa-edit-content-sections-new-tab-item_2' %}
        {% block content %}
            Contents of custom secondary section
        {% endblock %}
    {% endembed %}
{% endblock %}
```

I don't think using SEO name was fortunate because SEO feature will be available in the next release and it'll be moved to `Meta` tab by default.